src/authentication/footer/styles.scss

```diff
@@ -0,0 +1,5 @@
+@import '../styles';
+
+.invite-footer {
+  @include footer();
```

Comment:
Is anything else using the `footer` mixin? If not, we should just put the styles here. The intent of the mixins is when multiple pages need the same styling but if all those pages are just gonna use the the same component then we don't need the mixin.

Reply:
Nice, I changed it to this from Ratiks suggestion but I'll re-add the styles here and remove the footer mixin. Thanks Dale!

Reply:
Yeah, there's just so many ways to share code it's hard to choose the "right" one. lol